fix: UTC-374: Export breaking when annotation result is None

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/19117714676)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/662
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19082838431

mcanu commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19117548254)
> Successfully merged:  create mode 100644 web/libs/ui/src/lib/enterprise-upgrade-overlay/index.ts

mcanu committed:
Merge branch 'develop' into 'fb-utc-374'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19117548254

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19117714676)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19117714676

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/19139797034
